### ISSUE TITLE
Running Maven Build but failed to find the geoff jar file.

### ISSUE THREAD

**Issue by harvey911** [jexp/neo4j-shell-tools#28]
[ERROR] Failed to execute goal on project import-tools: Could not resolve dependencies for project org.neo4j.shell:import-tools:jar:2.0-SNAPSHOT: Failure to find com.nigelsmall:geoff:jar:0.5.0 in http://m2.neo4j.org/content/repositories/snapshots/ was cached in the local repository, resolution will not be reattempted until the update interval of neo4j has elapsed or updates are forced -> [Help 1]

**Comment by Raynooo** [comment]
Had the same problem. I downloaded the file from there 
https://github.com/neo4j-contrib/m2/blob/master/releases/com/nigelsmall/geoff/0.5.0/geoff-0.5.0.jar.md5

And then had to enter this command to install the package to the local repository manually :
mvn install:install-file -Dfile=geoff-0.5.0.jar -DgroupId=com.nigelsmall -DartifactId=geoff -Dpackaging=jar -Dversion=0.5.0

Hope this helps you. I guess this is because the package could not be found in the maven repository ?

**Comment by gquercini** [comment]
I had the same issue too. 
To solve the issue, I added to pom.xml the repositories indicated at this URL:

https://github.com/neo4j-contrib/m2

Now it compiles just fine, but I get an error in tests. 

Tests in error: 
  testRunWithInputFile2(org.neo4j.shell.tools.imp.ImportCypherAppTest): Cannot open file import.csv for reading.

It seems that this file import.csv does not exist.

Best

**Comment by Raynooo** [comment]
Yeah this test seems to fail because the input file simply doesn't exist. If you look at the other tests (especially the first testRunWithInputFile) you'll notice they all use another test file (in.csv) which is always created and therefore exists. 

To bypass it you can simply remove the @Test annotation on line 93 or you'll have to correct it (most likely using the createFile() function with the appropriate parameters).

**Comment by gquercini** [comment]
Thanks for the hint Raynoo.

**Comment by oguennec** [comment]
Following Raynooo comment from 11 Jul 2014, I have successfully added geoff-0.5.0.jar to local maven repository as follows:

git clone https://github.com/neo4j-contrib/m2.git
cd neo4j_contrib/m2/releases/com/nigelsmall/geoff/0.5.0
cp geoff-0.5.0.pom pom.xml
mvn clean install:install-file -Dfile=geoff-0.5.0.jar -DgroupId=com.nigelsmall -DartifactId=geoff -Dpackaging=jar -Dversion=0.5.0

I could then compile and generate jar (by skipping tests) after adding missing dependencies (commons-lang, jackson-mapper-asl). See szarnyasg pull request on 18 Jan and comments.
